app/js/track_events.js

```diff
@@ -0,0 +1,42 @@
+$(function() {
+  'use strict';
+
+  trackMenuClick();
+
+  analytics.trackForm($('.Nav form'), 'Subscribed Newsletter in the menu', getFormProperties);
+  analytics.trackForm($('.Newsletter form'), 'Subscribed Newsletter in the main content', getFormProperties);
+
+  $('[data-track-link]').each(function() {
+    var $element = $(this);
+    analytics.trackLink($element, $element.data('trackLink'));
+  });
+
+  $('[data-track-link-mailto]').each(function() {
+    var $element = $(this);
+    analytics.trackLink($element, $element.data('trackLinkMailto'), getMailFromMailTo);
+  });
+
+  function trackMenuClick() {
+    $('#BurgerBtn').on('click', function(event) {
+      var $btn = $(event.target);
+      if ($btn.hasClass('BurgerBtn--open')) {
+        analytics.track('Opened main menu');
+      } else {
+        analytics.track('Closed main menu');
+      }
```

Comment:
the scenario where [this](https://github.com/groupbuddies/rubyconfpt-2015/blob/master/app/js/nav.js#L24) for some reason runs after your function.

If I understood this correctly, your function requires the one I linked to be ran before, otherwise the events will be reversed (i.e. it will track an "Opened main menu" when it's being closed)

I was asking if there was no way to prevent this dependency. For example, using the `setTimeout(..., 0)` trick